src/loaders/svg/tvgSvgPath.cpp

```diff
@@ -525,6 +526,7 @@ bool svgPathToTvgPath(const char* svgPath, Array<PathCommand>& cmds, Array<Point
     while ((path[0] != '\0')) {
         path = _nextCommand(path, &cmd, numberArray, &numberCount);
         if (!path) break;
+        if (cmd == 0 || !strchr(commandList, cmd)) break;
         _processCommand(&cmds, &pts, cmd, numberArray, numberCount, &cur, &curCtl, &startPoint, &isQuadratic);
```

Comment:
Why not (!processCommand(...)) break;
It looks processCommand() can return false when not matched any case.

Reply:
you are right. i updated patch. thank you